### ISSUE TITLE
2 Questions on article

### ISSUE THREAD

**Issue by raelb** [johndatserakis/koa-vue-notes-web#6]
Hi,
Thanks for this nice project.
I had 2 queries on the article you wrote here:
https://vuejsdevelopers.com/2018/09/04/comparing-vue-and-react/

> Although Bootstrap usage in my apps is becoming less and less

I was curious to know what css libraries you are using more and more? :)

> After dealing with Vue for a few years and then working with React I quickly felt like Michael looking at Toby thinking, "Why are you the way you are?".

I was not sure which direction you meant this, and what you were referring to?

Thanks

**Comment by johndatserakis** [comment]
Hey there, thank you for checking out the project.

As far as what css libraries I'm using - really what I'm doing more and more is creating my own components. When I'm using Vue, I use the excellent `bootstrap-vue` library which basically has any really advanced component I may need. I've checked out libraries like `ant` and `element`, but I see that they really don't have mobile views in mind, so I stay away from those. So really what I'm doing is making my own agnostic components, and if the need is sufficiently advanced, I'll jump to a component that is already available in `bootstrap-vue`. The process has served me well so far.

Ha well about that other quote - it's from the show `The Office` when Michael just hates Toby and everything he represents. Now - my feeling towards React is not that bad at all - it's just referring to what a lot of people who have used both frameworks mention - the fact that Vue give you more of a "batteries-included" feel when building out a project. With React I felt like I was writing more boilerplate code and getting less actual work done - and not to mention that I found `redux` to be much more convoluted to use compared to `vuex`. Of course - these are just my experiences and others may have a different story to tell after using both.
